gapic/schema/wrappers.py

```diff
@@ -489,17 +528,34 @@ def idempotent(self) -> bool:
         return bool(self.options.Extensions[annotations_pb2.http].get)
 
     @property
-    def lro(self) -> bool:
-        """Return True if this is an LRO method, False otherwise."""
-        return getattr(self.output, 'lro_response', None)
+    def ident(self) -> metadata.Address:
+        """Return the identifier data to be used in templates."""
+        return self.meta.address
+
+    @utils.cached_property
+    def paged_result_field(self) -> Optional[Field]:
+        """Return the response pagination field if the method is paginated."""
+        # If the request field lacks any of the expected pagination fields,
+        # then the method is not paginated.
+        for page_field in ((self.input, int, 'page_size'),
+                           (self.input, str, 'page_token'),
+                           (self.output, str, 'next_page_token')):
+            field = page_field[0].fields.get(page_field[2], None)
+            if not field or field.type != page_field[1]:
+                return None
+
+        # Return the first repeated field.
+        for field in self.output.fields.values():
+            if field.repeated and field.message:
+                return field
```

Comment:
will this method implicitly return `None` if there are no `repeated` fields in the output message? can we return `None` explicitly to make it clear?

Reply:
Yes, and sure. Will do that before merging.

Reply:
Done. Thanks.